applications/endoscopy_tool_tracking_distributed/python/README.md

```diff
@@ -0,0 +1,67 @@
+# Distributed Endoscopy Tool Tracking
+
+Similar to the Endoscopy Tool Tracking application, the distributed version divides the application into three fragments:
+
+1. Video Input: get video input from an AJA card, a Yuan card or a pre-recorded video file.
+2. Inference: run the inference using LSTM and run the post-processing script.
+3. Visualization: display input video and inference results.
+
+Based on an LSTM (long-short term memory) stateful model, these applications demonstrate the use of custom components for tool tracking, including composition and rendering of text, tool position, and mask (as heatmap) combined with the original video stream.
+
+### Requirements
+
+- Python 3.8+
+- The provided applications are configured to either use the AJA or Yuan capture cards for an input stream, or a pre-recorded endoscopy video (replayer). 
+Follow the [setup instructions from the user guide](https://docs.nvidia.com/clara-holoscan/sdk-user-guide/aja_setup.html) to use the AJA capture card.
+
+### Data
+
+[📦️ (NGC) Sample App Data for AI-based Endoscopy Tool Tracking](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+The data is automatically downloaded and converted to the correct format when building the application.
+If you want to manually convert the video data, please refer to the instructions for using the [convert_video_to_gxf_entities](https://github.com/nvidia-holoscan/holoscan-sdk/tree/main/scripts#convert_video_to_gxf_entitiespy) script.
+
+### Run Instructions
```

Comment:
Can this run in the HoloHub container with `./dev_container build_and_run`?

Reply:
Yes, updated